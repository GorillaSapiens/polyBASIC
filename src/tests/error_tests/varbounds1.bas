a(22,5) = 1
