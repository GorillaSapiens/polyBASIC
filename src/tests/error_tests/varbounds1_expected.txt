SOURCE 1:1, VARIABLE ❮a❯ DIM1 AS 22 OUTSIDE BOUNDS1 0..10 FROM LINE 1
